### ISSUE TITLE
Launcher is not clickable using Vimium bindings in Firefox

### ISSUE THREAD

**Issue by heatherbooker** [dharness/react-chat-window#83]
When using Vimium bindings and pressing `f`, I think the launcher should become an option. I imagine this also impacts accessibility. 

Not sure if this is a demo-specific issue or should be baked in to the Launcher icon.

**Comment by dharness** [comment]
Can you make a more general claim about what doesn't work? I'm not farmilliar with vimium

**Comment by heatherbooker** [comment]
I think the launcher/close icon should be a button.

**Comment by dharness** [comment]
I think this would be a nice to have - let's keep it on the back burner.

**Comment by dharness** [comment]
I notice you actually can't do this on intercom.io either
